services/webnn/ort/graph_builder_ort.cc

```diff
@@ -130,6 +130,9 @@ constexpr char kOpTypeWhere[] = "Where";
 constexpr char kInserted[] = "Inserted";
 constexpr char kUnderscore[] = "_";
 
+constexpr std::array<uint32_t, 3> kRznToZrnPermutation = {1, 0, 2};
+constexpr std::array<uint32_t, 4> kIfgoToIofgPermutation = {0, 3, 1, 2};
```

Comment:
Nit: please declare the constant close within the method that uses it. I don't think they are shared by other methods.

Reply:
Done. Please take another look, thanks a lot!